### DISCUSSION TITLE
Updating Docs: CLI commands

### DISCUSSION THREAD

**Discussion by learning-more** [AzuraCast/AzuraCast#4277] [Q&A] [answered]
Hi, 

I have a change for the documentation. I received an error when attempting to sign into the wiki.js docs site, so I am suggesting it in this discussion forum.

Page I'm referring to in the docs: `https://docs.azuracast.com/en/administration/command-line-interface`.

The current documentation notes the path for the cli.php file in Ansible installations is:
`php /var/azuracast/www/util/cli.php [command]`

However, I found the correct command posted by the awesome SilverEagle is actually:
`php /var/azuracast/www/bin/console [command]`

I verified this in my rolling release install, which is: `Rolling Release #db8bc3d (2021-06-10 20:32)`.

**Comment by BusterNeece** [comment, marked as answer]
Updated! Thanks for the tip. :)
